src/main/resources/avro/bdg.avdl

```diff
@@ -780,39 +780,189 @@ enum Allele {
 record GenotypeAnnotation {
 
   /**
+   Per-sample total alternate read depth. VCF genotype reserved key AD, Number=R, split for
+   multi-allelic sites into single integer values for the reference allele
+   (referenceReadDepth) and the alternate allele (alternateReadDepth, this field).
+   */
+  union { null, int } alternateReadDepth = null;
```

Comment:
I will need some help thinking about what these Number=R values on `Genotype`s mean and how they should be split.  I wrote the doc here to match that in `Variant` but am not sure it makes sense.